README.md

```diff
@@ -107,6 +107,22 @@ If requests are passed to `QBWC.add_job`, any `QBWC::Worker#requests` method wil
 
 Similarly, a `QBWC::Worker#handle_response` method cannot access variables that are in-memory at the time that `QBWC.add_job` is called; however, you can optionally pass a serializable value (for example, String, Array, or Hash) to `QBWC.add_job`. This data will immediately be persisted by `QBWC.add_job`, then later passed to `QBWC::Worker#handle_response` during a QuickBooks Web Connector session.
 
+### Sessions ###
+
+In certain cases, you may want to perform some initialization prior to each QuickBooks Web Connector session (a QuickBooks Web Connector session is established when you manually run (update) an application's web service in QuickBooks Web Connector, or when QuickBooks Web Connector automatically executes a scheduled update). For this purpose, you may optionally provide an initialization block that will be invoked once when each QuickBooks Web Connector session is established, and prior to executing any queued jobs.
+
+You assign this initialization block prior to any QuickBooks Web Connector session being established, outside of any `QBWC::Worker` classes. For example:
```

Comment:
OK sorry. In my view, a session initializer has more in common with application code than with configuration of the plumbing (whereas an authenticator seems the opposite), which suggests that a typical usage would be from a model or model initializer. This also suggests that a block might be a better construct than a Proc. Am I off-base?

Is there some technique that allows either approach? (assignment using Proc in config/initializers, or `set_session_initializer` block elsewhere, but not both)